### ISSUE TITLE
Visual Studio 2017 compatibility for UWP

### ISSUE THREAD

**Issue by endyb66** [sshnet/SSH.NET#419]
I can't get it to work on UWP platform in Visual Studio 2017 with latest updates. After the connect() call, i get the following exception

`Server response does not contain SSH protocol identification`

So why does this happen? Here is my code for the connection

```
using (var client = new SshClient(hostname, port, username, password))
{
    client.Connect();
    client.RunCommand("ls");
    client.Disconnect();
}
```

On the server side i get the following inside the logfile

`Did not receive identification string from xxx.xxx.xxx.xxx port xxxxx`

**Comment by ghost** [comment]
have the same problem

**Comment by piresbruno** [comment]
Having the same issue, does anyone now a workaround?
If i write a unit test, it works, if i use the same code on a UWP app, i get that error.

**Comment by piresbruno** [comment]
Downloaded the source code, used the development branch, added it to my project and it's working now.

**Comment by drieseng** [comment]
This is fixed in [2020.0.0-beta1](https://github.com/sshnet/SSH.NET/releases/tag/2020.0.0-beta1).
